app/models/manageiq/providers/vmware/infra_manager.rb

```diff
@@ -671,16 +671,22 @@ def assign_ems_created_on(vm_ids)
 
       # Of the VMs without a VM create time, filter out the ones for which we
       #   already have a VM create event
+      events = EmsEvent
+        .where(
+          :vm_or_template_id => vms_to_update.map(&:id),
+          :event_type        => ["VmCreatedEvent", "VmDeployedEvent"])
```

Comment:
@Fryguy looks like you have an extra `)`

Reply:
Gah...typo :(